components/typography/TextLink.js

```diff
@@ -0,0 +1,20 @@
+
+export default function TextLink({
+    href,
+    className,
+    target = '_self',
```

Comment:
`_self` is default. If we pass `undefined` to the `target` react will skip that attribute, so you can change it to the:

```suggestion
    target,
```

:)